setupext.py

```diff
@@ -28,7 +29,7 @@
 LOCAL_FREETYPE_HASH = '348e667d728c597360e4a87c16556597'
 
 if sys.platform != 'win32':
-    if sys.version_info[0] < 3:
+    if not PY3:
```

Comment:
Thanks! I left unchanged the meaning of the definition of PY3: it was (sys.version_info[0] >= 3), and it is now (sys.version_info.major >= 3), so the special case condition would be true for Python 2. Of course, I agree that it would be better to rename PY3 to PY3min following the PY32min criteria...